website/blog/2017-04-10-blog-post-two.md

```diff
@@ -0,0 +1,18 @@
+---
+title: New Blog Post
```

Comment:
Why is this boilerplate in here?

Reply:
refer to https://github.com/Microsoft/fast-dna/pull/1175#discussion_r237646930